man/man5/zfs-module-parameters.5

```diff
@@ -250,6 +250,25 @@ Use \fB1\fR for yes (default) and \fB0\fR for no.
 .sp
 .ne 2
 .na
+\fBparallel_vdev_open\fR (bool)
```

Comment:
For consistency with the other options how about we call this `zfs_vdev_parallel_open_enabled`.